lib/arrays.md

```diff
@@ -5,44 +5,65 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is a list of items.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+Team rosters would work well in an array, as would an ingredient list.
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+If your array is simply
+Days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday']
+#you can simply
+Days[whatver day you want to call]
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+Sure, with this method, Ruby would count down, starting at 0 then negative, backwards around the array until the right one.
 
-# How would you perform an operation on every element inside an Array?
+# ?How would you perform an operation on everey lement inside an Arrayy 
 
-Replace me with your response...
+I am really struggling with this.... am I trying to add bullets to a list of names?  or make a list of numbers to perform mathmatical opperations on?
```

Comment:
Ok, so if I want to display the array as a list, all I need to do is 'puts scores', or 'puts names', but although I remember this being a task I remember doing for the pre test, I can not find how to put a string in front of the array when displaying it, or how to put 'score' in front of the list of scores.  I can make an array, change the elements, delete them, display the list, but I can not find where I can put the 'score' in front of every element!